apps/site-nft/src/components/AddAccount/Connect/index.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react';
+import { Form } from 'semantic-ui-react';
+import { ConnectWeb3 } from '@thecointech/shared/containers/Account/Web3';
```

Comment:
You might want to catch the latest version of all those files. The changes are mainly styles so not a big deal